docs/core/deploying/trim-self-contained.md

```diff
@@ -73,8 +73,8 @@ If we're using .Net 5.0, we can update our project file to include the following
 
 Trim your application using the [dotnet publish](../tools/dotnet-publish.md) command. When you publish your app, set the following properties:
```

Comment:
Should this line be updated? It makes it sound like I should only change my project when I pubilsh.

```suggestion
Trim your application using the [dotnet publish](../tools/dotnet-publish.md) command. Set the following properties in your project file, then publish the app using `dotnet publish`.
```